Sources/SwiftDocC/Semantics/Metadata/PageImage.swift

```diff
@@ -11,19 +11,26 @@
 import Foundation
 import Markdown
 
+/// A directive to adjust the image associated with the page.
+///
+/// This directive can be used to control the image used both when rendering an icon representing the page such as
+/// in the navigator sidebar, and when rendering a card representing a page, e.g. in a  Topics section styled as a grid.
 public final class PageImage: Semantic, AutomaticDirectiveConvertible {
     public let originalMarkup: BlockDirective
     
+    /// The purpose for which to use this iamge.
     @DirectiveArgumentWrapped
     public private(set) var purpose: Purpose
     
+    /// The path at which to find the image resource.
```

Comment:
I think the word "path" here is incorrect. The other image source values are just the name of the image asset.  See [the @Image documentation](https://www.swift.org/documentation/docc/image)

```suggestion
    /// The name and file extension of an image in your DocC catalog.
```